This patch is required for Makefile-based builds of NSS 3.45 on armhf-linux.

Taken from upstream bug tracker:
https://bugzilla.mozilla.org/show_bug.cgi?id=1571316

diff --git a/nss/lib/freebl/ecl/curve25519_32.c b/nss/lib/freebl/ecl/curve25519_32.c
--- a/nss/lib/freebl/ecl/curve25519_32.c
+++ b/nss/lib/freebl/ecl/curve25519_32.c
@@ -29,6 +29,10 @@
  *  1. Convert custom integer types to stdint.h types
  */
 
+#ifdef FREEBL_NO_DEPEND
+#include "../stubs.h"
+#endif
+
 #include "ecl-priv.h"
 
 /* fe means field element. Here the field is \Z/(2^255-19). An element t,

